apps/design-system/components/colors.tsx

```
@@ -4,6 +4,49 @@ import color from 'ui/src/lib/tailwind-demo-classes'
+  x,
+  definition,
+}: {
+  x: string
+  definition: 'background' | 'border' | 'text' | 'colors' | 'palletes'
+}) => {
+  switch (definition) {
+    case 'background':
+      return (
+        <div className={cn(x, 'relative w-full h-12 border border-overlay rounded-full')}></div>
+      )
+      break
```

Biome (2.4.4)

[error] This code will never be reached ...

(lint/correctness/noUnreachable)